src/main/java/org/osc/controller/nsc/entities/InspectionHookEntity.java

```diff
@@ -0,0 +1,97 @@
+package org.osc.controller.nsc.entities;
+
+import static javax.persistence.FetchType.LAZY;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+
+import org.hibernate.annotations.GenericGenerator;
+
+@Entity
+public class InspectionHookEntity {
+
+	private String hookId;
+
+	@Column(name = "inspectedPortId")
+	private NetworkElementEntity inspectedPort;
+
+	@Column(name = "inspectionPortId")
+	private InspectionPortEntity inspectionPort;
+	private Long tag;
+	private Long hookOrder;
+	private String encType;
+	private String failurePolicyType;
+
+	public InspectionHookEntity() {
+	}
+
+	@Id
```

Comment:
to be consistent with OSC, we should use the annotations on the fields not the setters/geters.